src/lib/exec.rs

```diff
@@ -41,6 +41,21 @@ pub struct InterpreterBuilder {
     global: Value,
 }
 
+fn exec_assign_op(op: &AssignOp, v_a: ValueData, v_b: ValueData) -> Gc<ValueData> {
```

Comment:
Should this be `Value` instead of `Gc<ValueData>` ?

Reply:
Missed that, but fixed now :)